sass/_Facet.scss

```diff
@@ -8,6 +8,13 @@
   position: relative;
 }
 
+// Needed because this facet embeds FacetRange
```

Comment:
I am having difficulty visualising the issue and why the border of the FacetRange does not work for the TimespanFacet. Would it be possible to add a screenshot in the jira?

Reply:
![screen shot 2019-01-14 at 10 15 34 am](https://user-images.githubusercontent.com/4923043/51121425-69b61100-17e5-11e9-8949-8fd8198a5101.png)
I think it was this issue

Reply:
Yep